doc/sphinx/source/env_variables.rst

```diff
@@ -33,6 +33,11 @@ pocl.
  CL_INVALID_VALUE. If clock_gettime is available, messages
  will include a timestamp.
 
+ The old way (setting POCL_DEBUG to 1) has been updated to support categories.
```

Comment:
Why couldn't we just print out prefixes and use the grep to filter out the interesting lines? This seems quite a lot of code for logging and I don't think it is worth the effort to not print the uninteresting ones as when one is debugging it should not be performance critical.